src/main/java/io/codelex/arrays/practice/Exercise6.java

```diff
@@ -0,0 +1,23 @@
+package io.codelex.arrays.practice;
+
+import java.util.Arrays;
+
+public class Exercise6 {
+    public static void main(String[] args) {
+
+        int[] array1 = new int[10];
+
+        for(int i = 0; i <array1.length; i++){
+            int randomNumber = (int) (Math.floor(Math.random() * 100) + 1);
```

Comment:
šeit labāk `Random.nextInt()`, vienkāršāka metode kur uzreiz atgriež int, no casting labāk censties izvairīties pēc iespējas in general

Reply:
Paldies par padomu! Izlaboju!